packages/stryker-webpack-transpiler/testResources/zeroConfig/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "webpackZeroConfig",
+  "version": "1.0.0",
+  "description": "A webpack 4 project with zero configuration",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack --mode development"
+  },
+  "keywords": [
+    "webpack 4"
+  ],
+  "author": "Sander Koenders",
+  "license": "MIT",
+  "devDependencies": {
+    "webpack": "^4.1.1",
```

Comment:
do we need dependencies here?

Reply:
Ofcourse we do, more dependencies is always better!